recipe/meta.yaml

```diff
@@ -39,17 +39,17 @@ outputs:
         - numpy >1.13
         - scikit-image >=0.13
         - scikit-learn
-        - scipy >=0.15
+        - scipy >=0.15,<1.4
         - statsmodels
         - ipyparallel
-        - sympy
+        - sympy <1.5
         - natsort
         - dask >=0.18
         - traits >=4.5
         - requests
         - tqdm >=0.4.9
         - python-dateutil >=2.5
-        - pint >0.8
+        - pint >0.8,<0.10
```

Comment:
Shall we remove `<0.10` and not run the test suite, since we know that the test suite is working fine?
For the record, some tests fail because with pint 0.9, we have `u`, while with pint 0.10, we have `μ`.

Reply:
I think that it is better to leave it as is, because otherwise, if somebody runs the tests locally, they'll fail.

Reply:
Well, I am fine with both.